common/lib/core-interfaces/api-report/core-interfaces.api.md

```diff
@@ -213,6 +213,19 @@ export const isFluidCodeDetails: (details: unknown) => details is Readonly<IFlui
 // @public
 export const isFluidPackage: (pkg: any) => pkg is Readonly<IFluidPackage>;
 
+// Warning: (ae-incompatible-release-tags) The symbol "Provider" is marked as @public, but its signature references "ProviderPropertyKeys" which is marked as @internal
+//
+// @public
+export type Provider<T = unknown> = Partial<Pick<T, ProviderPropertyKeys<T>>>;
```

Comment:
I'm considering renaming this to FluidObject, rather than Provider as i think it may make the transition easier for people.
 Open to opinions